pages/index.js

```diff
@@ -52,13 +67,40 @@ const StyledNav = styled.nav`
   line-height: 1.2rem;
 `;
 
+const StyledBackButton = styled.input`
+  background-color: transparent;
+  border: none;
+  text-decoration: none;
+  color: black;
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 8.5px;
+  border: 1px solid black;
+  text-align: center;
+  background-color: lightskyblue;
+`;
+
+const StyledMessage = styled.p`
+  align-self: center;
+  text-align: center;
+  font-weight: 600;
+  margin: 1rem auto;
+`;
+
 export default function HomePage({ onAddEmotionEntry }) {
+  const [tension, setTension] = useState("0");
+  const [show, setShow] = useState(false);
+  const [id, setId] = useState();
+
   function handleSubmit(event) {
     event.preventDefault();
-
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
-    onAddEmotionEntry(data);
+
+    const id = uid();
```

Comment:
⚠️ Using the `id` variable within the `handleSubmit` function is potentially problematic as it hides the id from the upper scope of the component - _the state variable._ 

This is known as shadowing, where a local variable in an inner scope has the same name as a variable in the outer scope. This can lead to confusion and errors. 

```suggestion
    const newId = uid();
```